torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2272,6 +2288,49 @@ def _make_tensor_helper(shape, low=None, high=None):
                        SkipInfo('TestCommon', 'test_variant_consistency_jit',
                                 device_type='cuda', dtypes=[torch.float16]),
                    )),
+    UnaryUfuncInfo('sign',
+                   ref=np.sign,
+                   dtypes=all_types_and(torch.bfloat16, torch.half),
+                   dtypesIfCPU=all_types_and(torch.bool, torch.bfloat16, torch.half),
+                   dtypesIfCUDA=all_types_and(torch.bool, torch.bfloat16, torch.half),
+                   skips=(
+                       # Reference: https://github.com/pytorch/pytorch/issues/41245
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_extremal',
+                                dtypes=[torch.bfloat16, torch.float16, torch.float32, torch.float64]),
+                       # >>> np.sign(True)
+                       # ufunc 'sign' did not contain a loop
+                       # with signature matching types dtype('bool') -> dtype('bool')
+                       SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_normal',
+                                dtypes=[torch.bool]),
+                   )),
+    UnaryUfuncInfo('sgn',
```

Comment:
Would you also fix the documentation for sgn:

https://pytorch.org/docs/master/generated/torch.sgn.html?highlight=sgn#torch.sgn

The documentation has a few issues:

- doesn't start by clearly summarizing what the function actually computes
- has "elemants" instead of "elements"
- the latex formatting of the equation is poor
- the example uses 'x' for a single tensor instead of 't', and it has no spacing around the equals operator

Reply:
> doesn't start by clearly summarizing what the function actually computes

I actually disagree. I think the current doc description for sgn is perfectly clear.